baystation12.dme

```diff
@@ -158,6 +158,7 @@
 #include "code\controllers\subsystems\lighting.dm"
 #include "code\controllers\subsystems\machinery.dm"
 #include "code\controllers\subsystems\mob.dm"
+#include "code\game\gamemodes\events\holidays\Easter.dm"

```

Comment:
Check the order of this. Did you recompile the code after adding this? Cause it should be way more down.